### PR TITLE
cleanup before release

### PR DESCRIPTION
**Pull request by jonkeane**

### CODE REVIEW

File: _pkgdown.yml [hunk 1]
```diff
@@ -250,6 +251,11 @@ reference:
   - temp.options
   - which
   - with-context-manager
+  - hasFunction
```
Comment (nealrichardson) [inline _pkgdown.yml]: Nah, give this `#' @keywords internal` and don't include it in the reference index.

File: NEWS.md [hunk 2]
```diff
@@ -1,569 +1,569 @@
...
-* Fix a similar yet unrelated bug in creating numeric and other types of variables with all-`NA` data.
-* Fix `streamRows()` for case when sending only one row (#253).
-* Internal: support for the "selected_array" method of multiple response calculation, deprecated since 1.20.0, has been removed.
-* Internal: `getDimTypes()` returns a richer set of cube dimension types differentiating multiple response from categorical array dimensions. 
-* Internal: Added support for `alias`, `description`, and `notes` on `VariableTuples`
-* `makeArrayGadget()` launches a an RStudio to help you build valid categorical arrays
+# crunch 1.22.3
+
+## Improved support for subtotals
+- CrunchCubes can now be displayed with subtotals on any axis (not just rows).
```
Comment (nealrichardson) [inline NEWS.md]: Why all of the `*` to `-`?

File: NEWS.md [hunk 3]
```diff
@@ -1,569 +1,569 @@
...
-* `changeCategoryID()` tries to unset then reset the dataset exclusion if that impacts its progress. Best practice is to disable exclusions before running `changeCategoryID()` if at all possible.
-* Setting the `ordering<-` of datasets within a project will now drop any invalid entries with a warning, rather than error.
-* Fix a bug introduced in 1.22.0 in creating categorical variables from factors with missing values.
-* Fix a similar yet unrelated bug in creating numeric and other types of variables with all-`NA` data.
-* Fix `streamRows()` for case when sending only one row (#253).
-* Internal: support for the "selected_array" method of multiple response calculation, deprecated since 1.20.0, has been removed.
-* Internal: `getDimTypes()` returns a richer set of cube dimension types differentiating multiple response from categorical array dimensions. 
-* Internal: Added support for `alias`, `description`, and `notes` on `VariableTuples`
-* `makeArrayGadget()` launches a an RStudio to help you build valid categorical arrays
+# crunch 1.22.3
```
Comment (nealrichardson) [inline NEWS.md]: Bump to 1.23.0?

File: NEWS.md (hunk 2 above)
Comment (jonkeane) [reply NEWS.md]: I took this directly from the news example from pkgdown, it doesn't actually make any difference either way so I've reverted.

**Pull request merged**